### DISCUSSION TITLE
What does killactive exactly do?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [hyprwm/Hyprland#8869] [Q&A]
What happens under the hood when I kill/close the window? I read this [section](https://wayland.app/protocols/wayland#wl_surface:request:destroy), but it is not what I'm looking for.

**Comment by vaxerski** [comment]
https://wayland.app/protocols/xdg-shell#xdg_toplevel:event:close

**Reply by azizsayfiddin** [reply]
> https://wayland.app/protocols/xdg-shell#xdg_toplevel:event:close

Is there any quit event, like one on every macOS app? I can use ctrl+q, since most linux apps support it, but some of them don't.

**Reply by vaxerski** [reply]
there is not, I guess you could send a SIGTERM to the pid.
